src/vsix/UI/ViewModels/StartViewModel.cs

```diff
@@ -15,12 +15,23 @@ public StartViewModel(RecentItemsViewModel recentItemsViewModel, StartItemsViewM
             Name = "Start";
             Title = GeneralOptions.Instance.StartTabTitleText;
             TabVisible = false;
-            Columns = new ObservableCollection<ColumnViewModel>
+            if (NewsItemsOptions.Instance.DisplayNewsItems)
             {
-                recentItemsViewModel,
-                startItemsViewModel,
-                newsItemsViewModel
-            };
+                Columns = new ObservableCollection<ColumnViewModel>
+                {
+                    recentItemsViewModel,
+                    startItemsViewModel,
+                    newsItemsViewModel
+                };
+            }
+            else
+            {
```

Comment:
I hadn't even thought to approach it this way. I think that's why I didn't finish it years ago. I just made the column invisible, and of course it was invisible, but was still there on the page.